HandlerLifecycleManager.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace Symfony\Bundle\MonologBundle;
+
+
+use Monolog\Handler\HandlerInterface;
+
+class HandlerLifecycleManager
```

Comment:
should be tagged as `@internal`. We really don't want to expose it in the BC promise.